rabbit-source/src/main/java/org/springframework/cloud/stream/module/rabbit/source/RabbitSourceProperties.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.rabbit.source;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Properties for the Rabbit Source.
+ *
+ * @author Gary Russell
+ *
+ */
+@ConfigurationProperties
+public class RabbitSourceProperties {
+
+	/**
+	 * Whether rejected messages should be requeued.
+	 */
+	private Boolean requeue;
+
+	/**
+	 * Whether the channel is transacted.
+	 */
+	private Boolean channelTransacted;
+
+	/**
+	 * The queues to which the source will listen for messages.
+	 */
+	private String queues;
```

Comment:
Haven't you tried with ` String[]`?
I agree about `"At least one queue is required"`, but seems for me the `ConversionService` can do its magic for us here.